paddle/framework/error.h

```diff
@@ -122,15 +117,13 @@ class Error {
    */
   bool isOK() const { return msg_ == nullptr; }
 
-  /**
-   * @brief check this status by glog.
-   * @note It is a temp method used during cleaning Paddle code. It will be
-   *       removed later.
-   */
-  void check() const { CHECK(this->isOK()) << msg(); }
-
-private:
+ private:
   std::shared_ptr<std::string> msg_;
```

Comment:
No. Because if we using `std:: string`, `Error` will copy by value each time, which is very time-consuming.